src/test/routes/receiver.ts

```diff
@@ -74,6 +75,7 @@ describe('Login receiver', async () => {
           draftStoreServiceMock.resolveFind('claim')
           claimStoreServiceMock.resolveRetrieveByClaimantIdToEmptyList()
           claimStoreServiceMock.resolveRetrieveByDefendantIdToEmptyList()
+          draftStoreServiceMock.resolveFindNoDraftFound()
```

Comment:
Here I can only guess the second one is to mock request for response draft but without looking at receiver code I would not be able to tell it. Please be more specific in what response is being mocked.